packages/embark-test-runner/src/index.js

```diff
@@ -91,6 +88,18 @@ class TestRunner {
     });
   }
 
+  generateCoverageReport() {
+    const coveragePath = path.join(dappPath(), ".embark", "coverage.json");
```

Comment:
```suggestion
    dappPath(".embark", "coverage.json");
```

Reply:
I forget we can use that format too. I’ll fix first thing in the am.